src/sass/champion_overwrite.scss

```diff
@@ -1,3 +1,7 @@
+html {
+    font-size: 62.5%; //to be remove once champion style fixes is ready
```

Comment:
we had to remove this because it was making the font in some other pages too small, so please make sure it doesn't break those pages anymore.